src/components/Firestore/Field/utils.ts

```diff
@@ -15,9 +15,18 @@
  */
 
 import { firestore } from 'firebase';
-import { FieldType } from './models';
 
-export function getFieldType(value: any): FieldType {
+import { FieldType, FirestoreAny } from './models';
+
+export function getParentPath(path: string[]) {
+  return path.splice(0, path.length - 1);
+}
+
+export function getLeafPath(path: string[]) {
```

Comment:
Consider renaming to `getLeafKey`? The returned value is a key (`string`) instead of path (`string[]`).

Reply:
Done